jax/random.py

```diff
@@ -371,17 +373,19 @@ def _shuffle(key, x, axis):
   return x
 
 
-def normal(key, shape, dtype=onp.float32):
+def normal(key, shape, dtype=onp.float64):
```

Comment:
nit: similar to above, can we have a default `shape=()` here?

Reply:
Done!